vernac/comAssumption.ml

```diff
@@ -10,59 +10,90 @@
 
 open CErrors
 open Util
-open Vars
 open Names
 open Context
 open Constrintern
 open Impargs
 open Pretyping
 
-module RelDecl = Context.Rel.Declaration
+module NamedDecl = Context.Named.Declaration
+
```

Comment:
```suggestion
```

Reply:
I think I find it clearer to skip a line here.